website/docs/integrator-section/transferable-record/overview.md

```diff
@@ -4,43 +4,43 @@ title: Transferable Record Overview
 sidebar_label: Overview
 ---
 
-Transferable Records are documents which extends on Verifiable Documents to allow a document to have an owner. These records references properties laid out in the [UNCITRAL Model Law on Electronic Transferable Records](https://www.uncitral.org/pdf/english/texts/electcom/MLETR_ebook.pdf)
+A Transferable Record is an extended form of a verifiable document, which allows the document to have an owner. Transferable Records reference the properties listed in [UNCITRAL Model Law on Electronic Transferable Records](https://www.uncitral.org/pdf/english/texts/electcom/MLETR_ebook.pdf).
```

Comment:
... reference the properties listed in **_the_** UNCITRAL Model ... 

In markdown it will show the words `UNCITRAL Model` instead of the link so having the "the" is correct I think

Reply:
added "the" back to the sentence.